documentation/source/development/design/labeling.rst

```diff
@@ -0,0 +1,166 @@
+| |feat:lightning|    | feat     | lightning     | light system                                                                       | #D4C5F9 |
++---------------------+----------+---------------+------------------------------------------------------------------------------------+---------+
+| |feat:logging|      | feat     | logging       | logging system                                                                     | #D4C5F9 |
++---------------------+----------+---------------+------------------------------------------------------------------------------------+---------+
+| |feat:octree|       | feat     | octree        | octree, cube computations                                                          | #D4C5F9 |
++---------------------+----------+---------------+------------------------------------------------------------------------------------+---------+
+| |feat:rendering|    | feat     | rendering     | rendering                                                                          | #D4C5F9 |
++---------------------+----------+---------------+------------------------------------------------------------------------------------+---------+
+| |feat:settings|     | feat     | settings      | settings                                                                           | #D4C5F9 |
++---------------------+----------+---------------+------------------------------------------------------------------------------------+---------+
+| |feat:shader|       | feat     | shader        | shaders                                                                            | #D4C5F9 |
++---------------------+----------+---------------+------------------------------------------------------------------------------------+---------+
+| |feat:texture|      | feat     | texture       | textures                                                                           | #D4C5F9 |
++---------------------+----------+---------------+------------------------------------------------------------------------------------+---------+
+| |plat:linux|        | plat     | linux         | Linux specific issue                                                               | #004E00 |
++---------------------+----------+---------------+------------------------------------------------------------------------------------+---------+
+| |plat:macos|        | plat     | macos         | MacOS specific issue                                                               | #004E00 |
++---------------------+----------+---------------+------------------------------------------------------------------------------------+---------+
+| |plat:windows|      | plat     | windows       | Windows specific issue                                                             | #004E00 |
++---------------------+----------+---------------+------------------------------------------------------------------------------------+---------+
```

Comment:
I think a more neutral color for the OS-labels would be better because it does not affect the severity of a bug.